package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mason-js-sdk",
-  "version": "0.0.3-alpha",
+  "version": "0.0.6-alpha",
```

Comment:
No need to increment the actual version here until it's been released. First release can be 0.1.0, and dev releases can use a separate counter like `0.1.1-dev1` and `0.1.1-dev2`

Reply:
👍

Reply:
👍